platform/ios/ios.cmake

```diff
@@ -1,7 +1,7 @@
 option(MBGL_WITH_IOS_CCACHE "Enable ccache for iOS" OFF)
 
 if(NOT DEFINED IOS_DEPLOYMENT_TARGET)
-    set(IOS_DEPLOYMENT_TARGET "9.0")
+    set(IOS_DEPLOYMENT_TARGET "11.0")
```

Comment:
We could likely go to 13 (based on https://github.com/maplibre/maplibre-gl-native/discussions/689)